app/infra/walletUpdateService/walletUpdateService.js

```diff
@@ -24,6 +31,10 @@ class WalletUpdateService {
   static listenToDownloadUpdate({ onDownloadUpdateCompleted }: { onDownloadUpdateCompleted: () => void }) {
     ipcRenderer.once(ipcConsts.DOWNLOAD_UPDATE_COMPLETED, () => {
       listenerCleanup({ ipcRenderer, channels: [ipcConsts.DOWNLOAD_UPDATE_COMPLETED] });
+      notificationsService.notify({
```

Comment:
What condition causes the update modal to be displayed? When it's the exact time to notify user that there is an update?

Reply:
This is the notification for download completed, not that an update is available.

Reply:
My question is not "what is this" but when "got update" modal is displayed and if it is consistent with notification and it's content

Reply:
the content of the notification is "App update downloaded." when the download completes. It is consistent.